### PR TITLE
Eos 24474 2:  Changed the PV and PVC storage capacity and volume paths

### PR DESCRIPTION
**Pull request by kailashyadav11**
Signed-off-by: Kailash Yadav <kailash.yadav@seagate.com>

# Problem Statement
- Problem statement

# Design
-  For Bug, Describe the fix here.
-  For Feature, Post the link for design

# Coding
   Checklist for Author
-  [x] Coding conventions are followed and code is consistent

# Testing 
  Checklist for Author
- [ ] Unit and System Tests are added
- [ ] Test Cases cover Happy Path, Non-Happy Path and Scalability
- [x] Testing was performed with RPM

# Impact Analysis
  Checklist for Author/Reviewer/GateKeeper
- [ ] Interface change (if any) are documented
- [ ] Side effects on other features (deployment/upgrade)
- [ ] Dependencies on other component(s)

# Review Checklist 
  Checklist for Author
- [x] JIRA number/GitHub Issue added to PR
- [x] PR is self reviewed
- [x] Jira and state/status is updated and JIRA is updated with PR link
- [x] Check if the description is clear and explained

# Documentation
  Checklist for Author
- [ ] Changes done to WIKI / Confluence page / Quick Start Guide

### CODE REVIEW

**Comment by cortx-admin** [comment]
Can one of the admins verify this patch?

**Review by nitin-seagate — APPROVED** [review]
LGTM

File: test/deploy/kubernetes/persistent-volumes/storage_devices_node1.yaml [hunk 1]
```diff
@@ -60,3 +60,24 @@ spec:
...
+  hostPath:
+    path: "/var/data/cortx"
+  nodeAffinity:
+    required:
+      nodeSelectorTerms:
+      - matchExpressions:
+        - key: node-name
+          operator: In
+          values:
+          - node1
```
Comment (nitin-seagate) [inline test/deploy/kubernetes/persistent-volumes/storage_devices_node1.yaml]: Add new line at EOF

File: test/deploy/kubernetes/persistent-volumes/storage_devices_node2.yaml [hunk 2]
```diff
@@ -60,3 +60,24 @@ spec:
...
+  hostPath:
+    path: "/var/data/cortx"
+  nodeAffinity:
+    required:
+      nodeSelectorTerms:
+      - matchExpressions:
+        - key: node-name
+          operator: In
+          values:
+          - node2
```
Comment (nitin-seagate) [inline test/deploy/kubernetes/persistent-volumes/storage_devices_node2.yaml]: Add new line at EOF

File: test/deploy/kubernetes/volume-claims/storage_volumes_node2.yaml [hunk 3]
```diff
@@ -1,41 +1,55 @@
...
+  name: local-node2
+  namespace: cortx
+spec:
+  storageClassName: storage-device
+  accessModes:
+  - ReadWriteOnce
+  resources:
+    requests:
+      storage: 1Gi
+  volumeName: local-node2
```
Comment (nitin-seagate) [inline test/deploy/kubernetes/volume-claims/storage_volumes_node2.yaml]: Add new line at EOF

File: test/deploy/kubernetes/volume-claims/storage_volumes_node3.yaml [hunk 4]
```diff
@@ -1,41 +1,55 @@
...
+  name: local-node3
+  namespace: cortx
+spec:
+  storageClassName: storage-device
+  accessModes:
+  - ReadWriteOnce
+  resources:
+    requests:
+      storage: 1Gi
+  volumeName: local-node3
```
Comment (nitin-seagate) [inline test/deploy/kubernetes/volume-claims/storage_volumes_node3.yaml]: Add new line at EOF

File: test/deploy/kubernetes/volume-claims/storage_volumes_node1.yaml [hunk 5]
```diff
@@ -1,41 +1,55 @@
...
+  name: local-node1
+  namespace: cortx
+spec:
+  storageClassName: storage-device
+  accessModes:
+  - ReadWriteOnce
+  resources:
+    requests:
+      storage: 1Gi
+  volumeName: local-node1
```
Comment (nitin-seagate) [inline test/deploy/kubernetes/volume-claims/storage_volumes_node1.yaml]: Add new line at EOF

**Review by nitin-seagate — CHANGES_REQUESTED** [review]

**Review by nitin-seagate — APPROVED** [review]
LGTM

**Review by 83bhp — APPROVED** [review]

**Pull request merged**